### PR TITLE
Subtitle conversion

### PR DESCRIPTION
**Pull request by tomaszzmuda**

### CODE REVIEW

File: Xabe.FFmpeg/Subtitle.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            _path = path;
+        }
+
+        /// <summary>
+        ///     Convert subtitle to specific format
+        /// </summary>
+        /// <param name="outputPath"></param>
+        /// <param name="subtitleFormat"></param>
+        /// <returns></returns>
```
Comment (lkischler) [inline Xabe.FFmpeg/Subtitle.cs]: ?!

File: Xabe.FFmpeg/Subtitle.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        public async Task<bool> Convert(string outputPath, SubtitleFormat subtitleFormat)
+        {
+            var codec = "";
+            switch(subtitleFormat)
+            {
+                case SubtitleFormat.SRT:
```
Comment (lkischler) [inline Xabe.FFmpeg/Subtitle.cs]: why not 'format extension' attribute on enum? this switch will be reduced

File: Xabe.FFmpeg/Subtitle.cs (hunk 1 above)
Comment (tomaszzmuda) [reply Xabe.FFmpeg/Subtitle.cs]: done

File: Xabe.FFmpeg/Subtitle.cs (hunk 2 above)
Comment (tomaszzmuda) [reply Xabe.FFmpeg/Subtitle.cs]: Changed it to description attribute

**Pull request merged**